ckanext/qa/i18n/en_AU/LC_MESSAGES/ckanext-qa.po

```diff
@@ -18,14 +18,6 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.3.4\n"
 
-#: ckanext/qa/reports.py:83 ckanext/qa/reports.py:89 ckanext/qa/reports.py:150
-msgid "total_stars"
-msgstr "total_ticks"
-
-#: ckanext/qa/reports.py:84 ckanext/qa/reports.py:90 ckanext/qa/reports.py:151
-msgid "average_stars"
```

Comment:
ok, so do we make it do what they wanted which is to swap starts for ticks.  Is there css in this plugin that does that?

Reply:
cool, so that needs to be updated from star back to tick for its variables